packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -91,6 +94,20 @@ export type ViewerOptions = Partial<
         }>
 >;
 
+/**
+ * Information computed from the hot spot surface data, canvas and mesh datas
+ */
+export interface HotSpotPositions {

```

Comment:
Any reason not to just use `HotSpot` here? I think it is fine to expose Babylon types directly.